common/src/main/java/org/alfresco/hxi_connector/common/model/ingest/IngestEvent.java

```diff
@@ -62,4 +63,13 @@ public static class ContentInfo implements Serializable
         private String encoding;
         private String mimetype;
     }
+    @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+    @Data
+    @Accessors(fluent = true)
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class AncestorsInfo implements Serializable
+    {
+        private String parentId;
```

Comment:
Why do we need a class for this rather than just a string? Incidentally I'd prefer extracting this to its own class, which will make all the references to it neater.